packages/core/saved-objects/core-saved-objects-api-browser/src/simple_saved_object.ts

```diff
@@ -49,6 +49,8 @@ export interface SimpleSavedObject<T = unknown> {
    * `namespaceType: 'agnostic'`.
    */
   namespaces: SavedObjectType<T>['namespaces'];
+  /** If the document is managed */
```

Comment:
```suggestion
/**
   * Flag indicating if a saved object is managed by Kibana (default=false)
   *
   * This can be leveraged by applications to e.g. prevent edits to a managed
   * saved object. Instead, users can be guided to create a copy first and
   * make their edits to the copy.
   */
```